Dockerfile

```diff
@@ -31,6 +32,11 @@ ARG no_update_flag
 
 # no_update_flag doesn't really work; .cache directory is re-created empty each time
 RUN SKIP_BUILD_UPDATE_CHECK=true ./build.sh $verbose_or_quiet_flag $no_update_flag && \
+    virtualenv ./env && \
+    source ./env/bin/activate && \
+    pip install lxml cssselect && \
+    python ./search_index.py && \
+    deactivate && \
```

Comment:
So, these should not go in the dockerfile, but instead in the build.sh script. That way they run both with and without docker.